superduper/components/component.py

```diff
@@ -351,7 +336,7 @@ def set_status(self):
         :param status: The status to set the component to.
         """
         return self.db.metadata.set_component_status(
-            self.__class__.__name__, self.uuid, status_update=ready_status()
+            self.__class__.__name__, self.uuid, status_update=running_status()
```

Comment:
This should be `ready_status()`

Reply:
I 've deprecated `ready` for `running`. This is to have consistent semantics across the framework.